src/main/scala/com/gu/facebook_news_bot/BotService.scala

```
@@ -0,0 +1,91 @@
+package com.gu.facebook_news_bot
+
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.actor.ActorSystem
+import akka.event.{Logging, LoggingAdapter}
+import akka.stream.{ActorMaterializer, Materializer}
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.amazonaws.services.dynamodbv2.document.DynamoDB
+import com.gu.facebook_news_bot.models.FromFacebook
+import com.gu.facebook_news_bot.services.{Capi, Facebook, UserStore}
+import com.typesafe.config.Config
+import de.heikoseeberger.akkahttpcirce.CirceSupport
+import io.circe.generic.auto._
+import util.AWSConstants._
+import com.gu.cm.{Configuration, Mode}
+import com.gu.facebook_news_bot.logic.StateHandler
+
+import scala.concurrent.ExecutionContextExecutor
+
+trait BotService extends CirceSupport {
+  implicit val system: ActorSystem
+  implicit def executor: ExecutionContextExecutor
+  implicit val materializer: Materializer
+  val logger: LoggingAdapter
+
+  val config: Config = Configuration("facebook-news-bot.properties", Mode.Dev).load
+
+  val stage = config.getString("stage")
+
+  val userStore = {
+    val usersTable = config.getString("aws.dynamo.usersTableName")
```

What does getString return, a String or an Option[String]? If it returns a String you may want to use a function that returns an Option[String] and fail fast (sys.error) on the cases where important config to the app is missing.